local/kind/ingress-router-us-east1.yaml

```diff
@@ -329,7 +329,7 @@ spec:
       containers:
         - name: controller
           image: k8s.gcr.io/ingress-nginx/controller:v1.0.3@sha256:4ade87838eb8256b094fbb5272d7dda9b6c7fa8b759e6af5383c1300996a7452
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: Always
```

Comment:
It is a change for no change. The behavior stays the same

Reply:
I see, but Checkov always reported `CKV_K8S_15: "Image Pull Policy should be Always"` when using`imagePullPolicy: IfNotPresent`.